educator/src/Dscp/DB/SQLite/Queries.hs

```diff
@@ -546,19 +546,27 @@ isAssignedToStudent student assignment = do
     |]
 
 data CourseDetails = CourseDetails
-    { cdCourseId :: Course
+    { cdCourseId :: Maybe Course
     , cdDesc     :: Text
     , cdSubjects :: [Id Subject]
     } deriving (Show, Generic)
 
+instance Default CourseDetails where
```

Comment:
Frankly speaking, i hate `Default` class because often it's really hard to guess (without checking the instance):
* That the instance exists (I usually check monoid instance first)
* What it its behaviour

So I'd better use `nullCourse = ...`.